lighthouse-core/test/audits/resource-summary-test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const ResourceSummaryAudit = require('../../audits/resource-summary.js');
+const assert = require('assert');
+const networkRecordsToDevtoolsLog = require('../network-records-to-devtools-log.js');
+
+/* eslint-env jest */
+
+describe('Performance: Resource summary audit', () => {
+  let artifacts;
+  let context;
+  beforeEach(() => {
+    context = {computedCache: new Map()};
+
+    artifacts = {
+      devtoolsLogs: {
+        defaultPass: networkRecordsToDevtoolsLog([
+          {url: 'http://example.com/file.html', resourceType: 'Document', transferSize: 30},
+          {url: 'http://example.com/app.js', resourceType: 'Script', transferSize: 10},
+          {url: 'http://third-party.com/script.js', resourceType: 'Script', transferSize: 50},
+          {url: 'http://third-party.com/file.jpg', resourceType: 'Image', transferSize: 70},
+        ])},
+      URL: {requestedUrl: 'https://example.com', finalUrl: 'https://example.com'},
+    };
+  });
+
+  it('has three table columns', () => {
+    return ResourceSummaryAudit.audit(artifacts, context).then(result => {
+      assert.equal(result.details.headings.length, 3);
+    });
+  });
+
+  it('includes the correct properties for each table item', () => {
+    return ResourceSummaryAudit.audit(artifacts, context).then(result => {
+      const item = result.details.items[0];
+      assert.equal(item.resourceType, 'total');
+      assert.ok(item.label);
```

Comment:
so we also have the jest [`expect` assertion library](https://jestjs.io/docs/en/expect) exposed, and it might be better to switch over to that for this file (apologies that the code base is split between `assert` and `expect` for tests).

Main thing it would buy here is that we added a helper for testing localized strings (it does `en-US` by default), so this could be `expect(item.label).toBeDisplayString('Total');`

([`network-request-test.js`](https://github.com/GoogleChrome/lighthouse/blob/f3b7429019883e811e6f07f78f36e9eb25cb76f6/lighthouse-core/test/lib/network-request-test.js) has a bunch of other nice examples of `expect()`, though the official docs are just fine since it's pretty much just an assertion library in the end :)